src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -574,9 +574,9 @@ CoilCoolingDXCurveFitSpeed &CoilCoolingDX::altModeNomSpeed()
 Real64 CoilCoolingDX::condMassFlowRate(bool const useAlternateMode)
 {
     if (useAlternateMode) {
-        return this->altModeNomSpeed().condenser_air_flow_rate;
+        return this->altModeNomSpeed().RatedCondAirMassFlowRate;
```

Comment:
Why is this one PascalCase while all others are camelCase?

Reply:
Good question. Probably copy/paste from the old DX coil. We can discuss cleaning these up next week.